CONTRIBUTING.md

```diff
@@ -61,6 +61,39 @@ This basically builds `terracognita` with the current sources.
 You also need to install other code dependencies not mandatory in the runtime environment:
   * [enumer](https://github.com/dmarkham/enumer) is used to generate some code.
   * [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) is used to format / organize code and imports. CI will perform a check on this, we highly recommend to run `$ goimports -w <your-modified-files>`.
+  * [mockgen](https://github.com/golang/mock) is used to generates the mocks for testing
+
+#### Add a new Provider
+
+:warning: It has to support `github.com/hashicorp/terraform-plugin-sdk@v2.0.0` or higher, providers with `v1` will not work anymore :warning:
+
+All the provider names of packages and so one are the ones used by the provider itself, the one in which all the resources are prefixed with (ex: `aws_instance` is `aws`)
```

Comment:
Couldn't that also be `registry.terraform.io/hashicorp/aws`, or that's only in cases of modules ?

Reply:
This one in particualar is the `provider.Provider.Source()`, the name is basically the resource prefix, so `aws`.